app/api/__init__.py

```diff
@@ -102,19 +102,20 @@
 # microlocations
 api.route(MicrolocationList, 'microlocation_list', '/microlocations', '/sessions/<int:id>/microlocations')
 api.route(MicrolocationDetail, 'microlocation_detail', '/microlocations/<int:id>',
-          '/events/<int:event_id>/microlocations', '/events/<identifier>/microlocations',
+          '/events/<int:event_id>/microlocations', '/events/<event_identifier>/microlocations',
```

Comment:
It says `microlocations` while it is associated with MicroLocationDetail. Can you please check?